docker/suricata/dist/update.sh

```diff
@@ -27,6 +27,22 @@ if [ "$myOINKCODE" != "" ] && [ "$myOINKCODE" == "OPEN" ];
 fi
 }
 
+function fuENRULES {
+  # Cleanup old files and extract new files.
+  rm -rf /tmp/rules /tmp/tpotce.rules
+  tar xfz /tmp/rules.tar.gz -C /tmp/ 2>&1 > /dev/null
+  # Create the new ruleset by:
+  # - looping through rule files, except deleted ones;
+  # - enabling all disabled rules (performance should be OK);
+  # - removing unnecessary empty/comment lines.
+  ls /tmp/rules/*.rules | grep -v deleted.rules | while read f;
+    do
+      cat $f | sed "s/^#alert/alert/" | grep -Ev "^(#|$)" >> /tmp/tpotce.rules
```

Comment:
I have left as-is the `sed` command that enables all commented-out rules (usually the performance-heavy ones) since you mentioned
> never had any issues with enabling all the rules

Another option would be to turn this into a configuration variable like `OINKCODE`